guides/common/modules/proc_upgrading-postgresql-on-the-external-database.adoc

```diff
@@ -0,0 +1,14 @@
+[id="Upgrading_PostgreSQL_on_the_External_Database_{context}"]
+= Upgrading PostgreSQL on the external database
+
+. If you use an external database, upgrade the external database to PostgreSQL 13.
+For more information, see {RHELDocsBaseURL}8/html/deploying_different_types_of_servers/using-databases#fast-upgrade-using-the-pg_upgrade-tool_migrating-to-a-rhel-8-version-of-postgresql[Fast upgrade using the pg_upgrade utility] in _Deploying different types of servers_.
```

Comment:
I think "Deploying different types of servers" isn't a very clear description. You're right it's the document title, but I wonder if we should be explicit that it's a RHEL 8 guide?

Reply:
In a number of other places, we specify the version of a RHEL guide a bit differently: in _{RHEL}{nbsp}X Guide Name_. Like here: https://github.com/theforeman/foreman-documentation/blob/8b1b052611eccfddfc4a615a2a745ca869e61e3b/guides/common/modules/ref_system-requirements.adoc?plain=1#L38  It would be better to stay consistent.